sdk/core/System.ClientModel/src/Message/BinaryContent.cs

```diff
@@ -191,4 +203,79 @@ public override void Dispose()
             }
         }
     }
+
+    private sealed class StreamBinaryContent : BinaryContent
+    {
+        private const int CopyToBufferSize = 81920;
+
+        private readonly Stream _stream;
+        private readonly long _origin;
+
+        public StreamBinaryContent(Stream stream)
+        {
+            if (!stream.CanSeek)
+            {
+                throw new ArgumentException("Stream must be seekable.", nameof(stream));
+            }
+
+            _stream = stream;
+            _origin = stream.Position;
+        }
+
+        public override bool TryComputeLength(out long length)
+        {
+            if (_stream.CanSeek)
```

Comment:
this will never be false, will it? ... you have this check in the ctor.

Reply:
Yep, I noticed this too.  This is copied from Azure.Core, but I think you're right that it can't change and isn't needed.